testnets/hypersigntestnet/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "hypersigntestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "hypersign",
+  "chain_id": "jagrat",
+  "bech32_prefix": "hid",
+  "daemon_name": "hid-noded",
+  "node_home": "$HOME/.hid-node",
+  "genesis": {
```

Comment:
can you please move this 'genesis' object inside the 'codebase' object?

Reply:
@JeremyParish69 genesis has been moved inside codebase